.github/workflows/daily-work.yaml

```diff
@@ -0,0 +1,47 @@
+name: Daily work
+
+on:
+  workflow_run:
+    workflows: ['e2e tests']
+    types: [completed]
+
+jobs:
+  build-operator:
```

Comment:
Rename it to "upload-operator-to-private-repo"

Reply:
Updated in https://github.com/vertica/vertica-kubernetes/pull/929/commits/c49e341eb3a46af8b91b01edcc634c0f284322d1